### PR TITLE
30 remove from favourites button functionality

### PR DESCRIPTION
**Pull request by ellaellac**
please review

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -51,7 +51,11 @@ const Card = ({ cardArray }) => {
             buttonText="Details"
             onClick={() => setActiveDetails(card)}
           />
-          <Button buttonName="remove" buttonText="Remove" />
+          <Button
+            buttonName="remove"
+            buttonText="Remove"
+            onClick={() => onClickPrimary(card._id)}
+          />
```
Comment (DanForder) [inline src/components/Card/Card.jsx:58]: Let's make this generic, the card shouldn't always have a remove button. What about the home page? That has a "save" button

File: src/context/UserContext.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const UserProvider = ({ children }) => {
   // login to default user
   // remove this when sign in functionality is complete
   useEffect(() => {
-    loginUser("barry@email.com", "password123!");
+    loginUser("test@test.com", "password123!");
```
Comment (DanForder) [inline src/context/UserContext.js:20]: undo this change

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/Card/Card.jsx (hunk 1 above)
Comment (ellaellac) [reply src/components/Card/Card.jsx:58]: done

File: src/context/UserContext.js (hunk 2 above)
Comment (ellaellac) [reply src/context/UserContext.js:20]: done

**Review by DanForder — APPROVED** [review]

**Review by ekauf — APPROVED** [review]
